src/System Application/App/Email/src/Connector/EmailRetrievalFilters.Codeunit.al

```diff
@@ -0,0 +1,109 @@
+namespace System.Email;
+
+/// <summary>
+/// This codeunit is used to store the filters for email connectors to use when retrieving emails.
+/// </summary>
+/// <remarks>
+/// Not all email connectors may use all the filters.
+/// </remarks>
+codeunit 4534 "Email Retrieval Filters"
```

Comment:
It would be easier if this was a table. You could have the default values and could skip calling Initialized. It is hard to read. it would also be more extensible.

Reply:
Initial design was a temporary table, but different from what you suggested from internal comms. 
Updated to use temp table which allows better extensibility now. 
Thanks!